### DISCUSSION TITLE
Always on Display widget moves around even when only 1 area box is ticked

### DISCUSSION THREAD

**Discussion by StevenUK1974** [NightscoutFoundation/xDrip#3428] [Bug reports]
Hi,

Version: 2024.04.09
Samsung S24
Libre Bluetooth (Libre 2 UK)

The Always on Display widget moves up and down the AOD screen even though I only have Top Centre ticked.

Thanks
Steven

**Comment by Navid200** [comment]
Are you sure this is not something that Samsung (your OS) is doing?
Can you post a screenshot of the classic status page?

@Der-Schubi

**Reply by StevenUK1974** [reply]
I'm not sure how to check that but my clock widget stays stationary.

Thanks
Steven


 ![Screenshot_20240411_173313_xDrip+.jpg](https://github.com/NightscoutFoundation/xDrip/assets/158083537/7b97abcb-76bf-48de-914a-83fd14084fc6)

**Comment by jamorham** [comment]
Does it move around within the top center area of the screen? Its supposed to move to avoid burn in.

**Reply by StevenUK1974** [reply]
Sorry, but I don't use this function anymore so I can't answer your question!

**Reply by Der-Schubi** [reply]
I think this is fixed by now, @Navid200 found a copy pasta error in my code some time ago, that caused at least a second part of the sreen to be used. I'm not on the PC right now to search for it.
